docs/features/global_tools.md

```diff
@@ -0,0 +1,177 @@
+# Pixi Global Tool Environment Installation
+
+With `pixi global`, users can manage globally installed tools in a way that makes them available from any directory.
+This means that the pixi environment will be placed in a global location, and the tools will be exposed to the system `PATH`, allowing you to run them from the command line.
+
+!!! note
+    The design for global tools is still in progress, and the commands and behavior may change in future releases.
+    The proposal for the global tools feature can be found [here](../design_proposals/pixi_global_manifest.md).
+
+## The Global Manifest
+Since `v0.33.0` pixi has a new manifest file that will be created in the global directory (default: `$HOME/.pixi/manifests/pixi-global.toml`).
+This file will contain the list of environments that are installed globally, their dependencies and exposed binaries.
+The manifest can be edited, synced, checked in to a version control system, and shared with others.
+
+A simple version looks like this:
+```toml
+[envs.vim]
+channels = ["conda-forge"]
+dependencies = { vim = "*" } # (1)!
+exposed = { vimdiff = "vimdiff", vim = "vim" } # (2)!
+
+[envs.gh]
+channels = ["conda-forge"]
+dependencies = { gh = "*" }
+exposed = { gh = "gh" }
+
+[envs.python]
+channels = ["conda-forge"]
+dependencies = { python = ">=3.10,<3.11" }
+exposed = { python310 = "python" } # (3)!
+```
+
+1. Dependencies are the packages that will be installed in the environment. You can specify the version or use a wildcard.
+2. The exposed binaries are the ones that will be available in the system path. `vim` has multiple so they will all be exposed.
+3. Here python is exposed as `python310` to avoid conflicts with other python installations. You can give it any name you want.
+
+### Channels
+The channels are the conda channels that will be used to search for the packages.
+There is a priority to these, so the first one will have the highest priority, if a package is not found in that channel the next one will be used.
+For example, running:
+```
+pixi global install --channel conda-forge --channel bioconda snakemake
+```
+Results in the following entry in the manifest:
+```toml
+[envs.snakemake]
+channels = ["conda-forge", "bioconda"]
+dependencies = { snakemake = "*" }
+exposed = { snakemake = "snakemake" }
+```
+
+More information on channels can be found [here](../advanced/channel_priority.md).
+
+### Exposed
+The exposed binaries are the ones that will be available in the system `PATH`.
+This is useful when the package has multiple binaries, but you want to get a select few, or you want to expose it with a different name.
+For example, the `python` package has multiple binaries, but you only want to expose the interpreter.
+Running:
+```
+pixi global expose add --environment python py3=python3
+```
+will create the following entry in the manifest:
+```toml
+[envs.python]
+channels = ["conda-forge"]
+dependencies = { python = ">=3.10,<3.11" }
+exposed = { py3 = "python3" }
+```
```

Comment:
Explain that you can run python by executing `py3`

Reply:
yes